ci/ci_openshift.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+TEST_RESULT=0
+
+function download_oc_client {
+  echo "==== Installing OC Client ===="
+  if [ -f ./oc ]; then
+    echo "oc client installed"
+  else
+    wget -q -N https://github.com/openshift/origin/releases/download/v1.3.0-alpha.2/openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
+    tar -zxf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
+    cp openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit/oc .
+    rm -rf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit
+    rm -rf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
+  fi
+}
+
+function start_cluster {
+  echo "==== Starting up cluster ===="
+  ./oc cluster up
+}
+
+function create_application {
+  echo "==== Creating Infinispan application ===="
+  ./oc new-app jboss/infinispan-server
```

Comment:
hmmm sounds interesting... have you tried to do that? If so, could you
please post an example?

On Tue, Aug 2, 2016 at 10:20 AM, Gustavo Fernandes <notifications@github.com
> wrote:

> In ci/ci_openshift.sh
> <https://github.com/jboss-dockerfiles/infinispan/pull/22#discussion_r73112477>
> :
>
> > +    wget -q -N https://github.com/openshift/origin/releases/download/v1.3.0-alpha.2/openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
> > +    tar -zxf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
> > +    cp openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit/oc .
> > +    rm -rf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit
> > +    rm -rf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
> > +  fi
> > +}
> > +
> > +function start_cluster {
> > +  echo "==== Starting up cluster ===="
> > +  ./oc cluster up
> > +}
> > +
> > +function create_application {
> > +  echo "==== Creating Infinispan application ===="
> > +  ./oc new-app jboss/infinispan-server
>
> Alternatively configure the ImageStream to pick local images as I wrote
> earlier
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jboss-dockerfiles/infinispan/pull/22/files/e962aa95b4b7432eb17ff3291a0ae297d60461ed#r73112477>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIBvhry4n9G4lAmfUHbOfWDeM_b6k3F7ks5qbv3rgaJpZM4JaTvU>
> .
>